pkg/utils/utils.go

```diff
@@ -217,6 +218,38 @@ func StringSliceMatch(slice []string, item string) bool {
 	return false
 }
 
+// Sift through a slice to determine if it contains eligible package
+// with optional version constrains.
+func SliceCheckString(slice []string, item string, item_version string) bool {
```

Comment:
```suggestion
func PackageSliceCheck(slice []string, item string, item_version string) bool {
```
`SliceCheckString` is a bit ambiguous if this is going to perform regex matches to validate packages and version constraints, I think we should aim for a function name that is closer to what this is doing.

Reply:
Good idea! Thanks and done!